src/Aspire.Dashboard/Components/Pages/Resources.razor.cs

```diff
@@ -196,6 +196,8 @@ protected override async Task OnInitializedAsync()
 
         _isLoading = false;
 
+        return;
+
```

Comment:
This doesn't do anything. Below it are just local methods.

Reply:
I prefer to keep the return before local methods for readability. But deleted